observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,72 @@ These release notes track the development of the Network Observability Operator
 
+[id="network-observability-operator-release-notes-1-6"]
+== Network Observability Operator 1.6.0
+The following advisory is available for the Network Observability Operator 1.6.0:
+// note to self: update errata live ID
+* link:https://access.redhat.com/errata/RHSA-2024:0853[Network Observability Operator 1.6.0]
+
+[id="network-observability-operator-1.6.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-lokiless-enhancements-1.6"]
+==== Enhanced use of Network Observability Operator without Loki
+You can now do more than ever with the Network Observability Operator by using Prometheus metrics and relying less on Loki for storage. For more information, see xref:../../observability/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki].
+
+[id="network-observability-custom-metrics-1.6"]
+==== Custom metrics API
+You can create custom metrics out of flowlogs data by using the `FlowMetrics` API. Flowlogs data can be used with Prometheus labels to customize cluster information on your dashboards. You can add custom labels for any subnet that you would want to identify in your flows and metrics. This enhancement can also be used to better identify traffic from or to IPs that are external to the cluster. For more information, see xref:../../observability/network_observability/metrics-alerts-dashboards.adoc#network-observability-custom-metrics_metrics-dashboards-alerts[Custom metrics] and <FLOW_METRICS_API_REF_LINK_HERE>
+
+[id="network-observability-eBPF-performance-enhancements-1.6"]
+==== eBPF performance enhancements
+With the following updates, you can experience improved performances of the eBPF agent, both in terms of CPU and memory: 
+
+* The eBPF agent now uses TCX webhooks instead of TC.
+* The *NetObserv / Health* dashboard integration with eBPF metrics allows you to view alerts in two cases: when the Network Observability eBPF agent hashmap table is full, which causes the eBPF agent processes flows with degraded performance, or when the capacity limiter is triggered. 
+* Duplicated flows are now merged into a list, which is tied to a single flow data, so Loki storage demand decreases significantly. Previously, duplicated flows were stored as individual flows, and flagged as duplicate if there were multiple of the same kind of flows. You could choose to show the duplicated flows using the *Duplicate flows* Query Option. Now instead of having multiple, individual flows, there is one flow with a list of matching *Interfaces* and *Interface directions*.
+
+[NOTE]
+===
+With the duplicated flows update, the *Interface* and *Interface Direction* fields in the *Network Traffic* table are renamed to *Interfaces* and *Interface Directions*, so any saved queries usi ng these fields need to be updated. 
```

Comment:
```suggestion
The duplicated flows update renames the *Interface* and *Interface Direction* fields in the *Network Traffic* table to *Interfaces* and *Interface Directions*. Update any saved queries that use these fields to... 
```
I don't know what the user does with these queries, but you could add that there.

Reply:
basically they need to add a "s" to the filter names `interface` => `interfaces` and `ifdirection` => `ifdirections`.
An example using a configured quick filter, if it was previously:
```yaml
    - filter:
        interface: genev_sys_6081
      name: genev
```

then it must be changed to (adding "s") :
```yaml
    - filter:
        interfaces: genev_sys_6081
      name: genev
```

Reply:
I updated this admonition to say the following:
[IMPORTANT]
===
With the duplicated flows update, the *Interface* and *Interface Direction* fields in the *Network Traffic* table are renamed to *Interfaces* and *Interface Directions*, so any bookmarked *Quick filter* queries using these fields need to be updated to `interfaces` and `ifdirections`. 
===